user_sync/app.py

```diff
@@ -148,7 +149,13 @@ def init_log(logging_config):
         if not os.path.exists(file_log_directory):
             os.makedirs(file_log_directory)
 
-        file_path = os.path.join(file_log_directory, datetime.date.today().isoformat() + ".log")
+	file_log_name = "{datestamp}.log".format(datestamp=datetime.date.today().isoformat())
+
+	if options['file_log_name']:
```

Comment:
This line is not necessary.  since you set the `file_log_name_format` in the builder, it will always be present.